scripts/Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM ubuntu:20.04
+FROM ubuntu:22.04
```

Comment:
`scripts/Dockerfile` does not seem to do anything, we can probably remove it. Was able to build docker containers and pass all tests without it.

Reply:
Agreed. I ran a successful build with this file deleted.